Mars-QA/Pages/Profile.cs

```diff
@@ -0,0 +1,235 @@
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Mars_QA.Pages
+{
+    public class Profile : CommonDriver
+    {
+        string actualMessage;
+        string actualMessageSkill;
+        string actualMessage1;
+        string actualMessage2;
+        public IWebElement addLangButton => dr.FindElement(By.XPath("//*[@id='account-profile-section']/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/thead/tr/th[3]/div"));
+        public IWebElement language=> dr.FindElement(By.XPath("//input[@placeholder='Add Language']"));
+        public SelectElement oSelect => new SelectElement(dr.FindElement(By.Name("level")));
+        public IWebElement AddButton => dr.FindElement(By.XPath("//input[@value='Add']"));
+       
+        public IWebElement SkillsTab => dr.FindElement(By.XPath("//a[normalize-space()='Skills']"));
+        public IWebElement addSkills => dr.FindElement(By.XPath("//*[@id='account-profile-section']/div/section[2]/div/div/div/div[3]/form/div[3]/div/div[2]/div/table/thead/tr/th[3]/div"));
+        public IWebElement skill => dr.FindElement(By.XPath("//input[@placeholder='Add Skill']"));
+        public SelectElement selectSkill => new SelectElement(dr.FindElement(By.XPath("//select[@class='ui fluid dropdown']")));
+        public IWebElement AddSkillButton => dr.FindElement(By.XPath("//*[@id='account-profile-section']/div/section[2]/div/div/div/div[3]/form/div[3]/div/div[2]/div/div/span/input[1]"));
+        public IWebElement languageTab => dr.FindElement(By.XPath("//*[@id='account-profile-section']/div/section[2]/div/div/div/div[3]/form/div[1]/a[1]"));
+        public IWebElement addNewLanguageButton => dr.FindElement(By.XPath("//*[@id='account-profile-section']/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/thead/tr/th[3]/div"));
+        public SelectElement languageType => new SelectElement(dr.FindElement(By.XPath("//*[@id='account-profile-section']/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/div/div[2]/select")));
+        public IWebElement messageBox => dr.FindElement(By.XPath("//div[@class='ns-box-inner']"));
+        public IWebElement skillmessagebox => dr.FindElement(By.XPath("//div[@class='ns-box-inner']"));
+        // public IWebElement Cancel => dr.FindElement(By.XPath("//input[@value='Cancel']"));
+        // public  IWebElement Hindi => dr.FindElement(By.XPath("//td[normalize-space()='Hindi']")); 
+        public IWebElement messageBox1 => dr.FindElement(By.XPath("//div[@class='ns-box-inner']"));
+        public IWebElement message => dr.FindElement(By.XPath("//div[@class='ns-box-inner']"));
+
+
+        public void AddLanguageAndSkill(string LanguageName, string LanguageType, string SkillName, string SkillType)
+        {
+
+            //Validate Empty Scenario by adding new language without filling fields
+            Thread.Sleep(3000);
+            addLangButton.Click();
+                 
+            Thread.Sleep(5000);
```

Comment:
why are you using thread sleep every where? you should make you of waits https://www.lambdatest.com/blog/explicit-fluent-wait-in-selenium-c/

Reply:
Ok , i will make changes.